manifest.yaml

```diff
@@ -107,7 +108,7 @@ interfaces:
       - http
 dependencies:
   lnd:
-    version: ">=0.12.1 <0.14.0"
```

Comment:
Does this version of TH stop working on LND versions < 0.14.2? @ProofOfKeags @elvece